Assets/ShooterProject/Scripts/Actors/AI/EnemyMovement.cs

```diff
@@ -96,11 +103,11 @@ private void Update()
 
 		private void ChangeBehaviour(EnemyBehaviour newBehaviour)
 		{
-			_behaviour = newBehaviour;
+			Behaviour = newBehaviour;
 			BehaviourChanged?.Invoke(newBehaviour);
 		}
 
-		private void OnTargetDied()
+		private void OnTargetDied(Health.Health targetHealth)
```

Comment:
```suggestion
		private void OnTargetDied(Health targetHealth)
```

У тебя же уже подключено пространство имен Health выше